homework01/rudolf/vector.h

```diff
@@ -0,0 +1,31 @@
+#ifndef __VECTOR_H_INCLUDED__   // if x.h hasn't been included yet...
+#define __VECTOR_H_INCLUDED__   //   #define this so the compiler knows it has been included
```

Comment:
We usually use `#pragma once`, it's shorter and you cannot have collisions on the name.